tests/integration/test_fim/test_registry/test_registry_report_changes/test_registry_file_size_values.py

```diff
@@ -70,53 +73,41 @@
 
 # Variables
 
-key = "HKEY_LOCAL_MACHINE"
-sub_key_1 = "SOFTWARE\\test_key"
-sub_key_2 = "SOFTWARE\\Classes\\test_key"
-
-test_regs = [os.path.join(key, sub_key_1),
-             os.path.join(key, sub_key_2)]
-
+test_regs = [os.path.join(WINDOWS_HKEY_LOCAL_MACHINE, MONITORED_KEY),
+             os.path.join(WINDOWS_HKEY_LOCAL_MACHINE, MONITORED_KEY_2)]
 test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
 wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
-reg1, reg2 = test_regs
-size_limit_configured = 10 * 1024
+scan_delay = 2
 
 # Configurations
 
-p, m = generate_params(modes=['scheduled'], extra_params={'WINDOWS_REGISTRY_1': reg1,
-                                                          'WINDOWS_REGISTRY_2': reg2,
+params, metadata = generate_params(modes=['scheduled'], extra_params={'WINDOWS_REGISTRY_1': test_regs[0],
+                                                          'WINDOWS_REGISTRY_2': test_regs[1],
                                                           'FILE_SIZE_ENABLED': 'yes',
-                                                          'FILE_SIZE_LIMIT': '10KB',
-                                                          'DISK_QUOTA_ENABLED': 'no',
-                                                          'DISK_QUOTA_LIMIT': '4KB'})
+                                                          'FILE_SIZE_LIMIT': '10KB'
+                                                        })
 
-configurations_path = os.path.join(test_data_path, 'wazuh_registry_report_changes_limits_quota.yaml')
+configurations_path = os.path.join(test_data_path, 'wazuh_registry_file_size_values.yaml')
 
-configurations = load_wazuh_configurations(configurations_path, __name__, params=p, metadata=m)
+configurations = load_wazuh_configurations(configurations_path, __name__, params=params, metadata=metadata)
 
 
 # Fixtures
 
-
 @pytest.fixture(scope='module', params=configurations)
 def get_configuration(request):
     """Get configurations from the module."""
     return request.param
 
 
-@pytest.mark.parametrize('size', [
-    (4 * 1024),
-    (16 * 1024),
+@pytest.mark.parametrize('size', [(4096), (16384)])
```

Comment:
Please add a comment explaining what these numbers mean. We know it, but this test must be clear to external teams.